packages/webdriverio/src/middlewares.js

```diff
@@ -1,6 +1,6 @@
-
 import refetchElement from './utils/refetchElement'
 import implicitWait from './utils/implicitWait'
+import { CustomRequestError } from 'webdriver/src/utils'
```

Comment:
it should be `'webdriver/build/utils'` instead of `'webdriver/src/utils'`

Reply:
Can we move `CustomRequestError` into `@wdio/utils` if it is used in multiple packages?

Reply:
I think it could be done in next PR